src/prompts/index.tsx

```diff
@@ -198,6 +204,55 @@ export class PromptManager {
         return false;
     }
 
+    private maybePersistReminder(prompt: PromptConfig) {
+        if (prompt.type !== 'alert') {
```

Comment:
Would it be more flexible here to use duck typing and just test if the declined config key exists? (thinking future if more types support it)

Reply:
Can't refactor just now as we take undefined for types that support it to mean suppressed.

Reply:
Had actually picked this up through changes made to the props intf which is now generic and provides the concrete prompt type config.

Reply:
ah yep, perfect